cmd/crc/cmd/start.go

```diff
@@ -63,6 +65,11 @@ func runStart(arguments []string) (*machine.StartResult, error) {
 		return nil, err
 	}
 
+	properties := telemetry.PropertiesFromContext(ctx)
+	for _, key := range []string{cmdConfig.CPUs, cmdConfig.Memory, cmdConfig.DiskSize} {
+		properties.Set(key, config.Get(key).AsInt())
```

Comment:
this assumes that the CPU, Memory and Disksize are always given. For these this might be the case, as they have default values... but might be risky for other options. Just saying

Reply:
Can we convert Memory and DiskSize to bytes before sending to telemetry?

Reply:
Yes very good point ! Will do that

Reply:
@guillaumerose @cfergeau what is the benefit of converting it to bytes, we are already sending the memory and disk as int as part of config or start option right?

Reply:
I was about to write it and I would have to remove the loop and convert manually here. Maybe it's better to keep what the user entered.